pinot-core/src/main/java/org/apache/pinot/core/operator/CombineOperator.java

```diff
@@ -168,13 +168,13 @@ public IntermediateResultsBlock callJob()
     try {
       mergedBlock = mergedBlockFuture.get(endTimeMs - System.currentTimeMillis(), TimeUnit.MILLISECONDS);
     } catch (InterruptedException e) {
-      LOGGER.error("Caught InterruptedException.", e);
+      LOGGER.error("Caught InterruptedException. (brokerRequest = {})", _brokerRequest, e);
```

Comment:
Do we need brokerRequest, or just requestId should be fine?

Reply:
I prefer brokerRequest. Otherwise, we need one more look up from the broker side, which will increase the time for debugging. Also, it's possible that broker side use the log sampling feature.

How do you think?

Reply:
Also, I checked the code and `brokerRequest` doesn't have the `requestId`. `instanceRequest` has the `requestId`. So, printing `brokerRequest` here looks like a better option to us.